databases/mysql8/Portfile

```diff
@@ -13,8 +13,8 @@ maintainers             {gmail.com:herby.gillot @herbygillot} \
                         openmaintainer
 
 # Set revision_client and revision_server to 0 on version bump.
-set revision_client     2
-set revision_server     2
+set revision_client     3
+set revision_server     3
```

Comment:
mysql8_server just installs server-related startup scripts and configuration. There's no reason to increase its revision for a change in the libraries. (But you can't change it back now.)